src/utils/columnUtils.ts

```diff
@@ -164,7 +164,7 @@ export function canEdit<R, SR>(column: Column<R, SR>, row: R): boolean {
   if (typeof column.editable === 'function') {
     return column.editable(row);
   }
-  return Boolean(column.editor ?? column.editor2 ?? column.editable);
+  return column.editable || (column.editor != null || column.editor2 != null) && column.editable !== false;
```

Comment:
A bit confusing with many or and and conditions.

```
if (typeof column.editable === 'function') return column.editable(row);
if (column.editable === false) return false;
return column.editor != null || column.editor2 != null;
```

Wdyt?

Reply:
This won't enable the default editor if `editable` is `true` and `editor` is nullish.
Any confusion could be addressed by having unit tests for this function.

Reply:
Are you thinking of using a default editor?

Reply:
https://github.com/adazzle/react-data-grid/blob/canary/src/editors/SimpleTextEditor.tsx
https://github.com/adazzle/react-data-grid/blob/canary/src/editors/EditorContainer.tsx#L161
We default to the `SimpleTextEditor` 🤔

Reply:
But only for the old editor. How would we handle it for the new editor once the old editor is removed?